cql3/Cql.g

```diff
@@ -1336,6 +1338,21 @@ listServiceLevelAttachStatement returns [std::unique_ptr<list_service_level_atta
       { $stmt = std::make_unique<list_service_level_attachments_statement>(); }
     ;
 
+/**
+ * opis
+ */
+describeStatement returns [std::unique_ptr<cql3::statements::raw::describe_statement> stmt]
+    @init {
+        bool fullSchema = false;
+        bool pending = false;
+        bool config = false;
+        bool only = false;
+    }
+    : ( K_DESCRIBE | K_DESC )
+    ( (K_KEYSPACES) => K_KEYSPACES       { $stmt = cql3::statements::raw::describe_statement::keyspaces(); }
```

Comment:
What does this weird `(K_KEYSPACES) => K_KEYSPACES` do that an ordinary `K_KEYSPACES` doesn't?

Reply:
I've copy-pasted it from Cassandra, but I couldn't find what is the difference while browsing the ANTLR's docs

Reply:
The joy of copy-pasting ;-)
I found an explantion. It turns out that the `... =>` is called "syntactic predicate", and it's needed in ANTLR with its very limited ability to parse two alternatives which share the same prefix. So for example if you have two targets "x" and  "y" which start with the same prefix, you can't just write `x | y` - Antlr will complain. If you do `(x) => x | y`, The `(x) => x` means it first tries to see if it can parse a whole x, and if not - don't try this option, try y instead. Why this is not the default behavior is a good question.

All that being said, I don't know why it's needed in this case... You can try writing simply `K_KEYSPACES` instead of `(K_KEYSPACES) => K_KEYSPACES` and see what Antlr complains - if at all.